packages/templates/cra-template-web-viewer/template/src/Auth.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import type { BrowserAuthorizationClientConfiguration } from "@itwin/browser-authorization";
+import { BrowserAuthorizationClient } from "@itwin/browser-authorization";
+
+export class Auth {
+  private static _client: BrowserAuthorizationClient;
+
+  public static initialize(
+    options: BrowserAuthorizationClientConfiguration
+  ): BrowserAuthorizationClient {
+    this._client = new BrowserAuthorizationClient(options);
```

Comment:
should you check if client is defined before setting it? same thing with the callback fn?

Reply:
Sure thanks